File: JavaScript/RecorridoDeMatrices.js
let matriz = [
    [1,2,3],
    [4,5,6],
    [7,8,9]
];

console.log("Recorrido por renglones:");
for(let i=0; i<matriz.length; i++) {
    let fila = "";
    for(let j=0; j<matriz[i].length; j++) {
        fila += matriz[i][j] + " ";
    }
    console.log(fila);
}

console.log("\nRecorrido por columnas:");
for(let j=0; j<matriz[0].length; j++) {
    let col = "";
    for(let i=0; i<matriz.length; i++) {
        col += matriz[i][j] + " ";
    }
    console.log(col);
}

File: Python/RecorridoDeMatrices.py
# Recorrido por renglones
matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

print("Recorrido por renglones:")
for fila in matriz:
    for valor in fila:
        print(valor, end=" ")
    print()

# Recorrido por columnas
print("\nRecorrido por columnas:")
for col in range(len(matriz[0])):
    for fila in range(len(matriz)):
        print(matriz[fila][col], end=" ")
    print()